itou/fixtures/django/05_test_users.json

```
@@ -1034,9 +1034,49 @@
       "public_id": "0ec9d5c3-01f2-479e-bd1b-5606991354ad",
       "title": "",
       "user_permissions": [],
-      "username": ""
+      "username": "test+cap@inclusion.beta.gouv.fr"
     },
     "model": "users.user",
     "pk": 28
+  },
+  {
+    "fields": {
+      "address_filled_at": null,
+      "address_line_1": "",
+      "address_line_2": "",
+      "ban_api_resolved_address": null,
+      "birthdate": null,
+      "city": "",
+      "coords": null,
+      "created_by": null,
+      "date_joined": "2023-06-21T13:49:22.935Z",
+      "department": "",
+      "email": "test+convergence@inclusion.beta.gouv.fr",
+      "external_data_source_history": null,
+      "first_login": null,
+      "first_name": "Gustave",
+      "geocoding_score": null,
+      "geocoding_updated_at": null,
+      "groups": [],
+      "has_completed_welcoming_tour": true,
+      "identity_provider": "IC",
+      "insee_city": null,
+      "is_active": true,
+      "is_staff": false,
+      "is_superuser": false,
+      "kind": "labor_inspector",
+      "last_checked_at": "2023-06-21T13:49:22.935Z",
+      "last_login": "2023-06-21T13:58:50.299Z",
+      "last_name": "Girondin",
+      "password": "pbkdf2_sha256$720000$wL3OTHgIZ7dHymGR8jXJ2k$5Z8vcMYeqYgbINE7bGzigH0rAMSaF2wkK7tYKTXH1e8=",
+      "phone": "",
+      "post_code": "",
+      "public_id": "67f4a2fc-36db-47d4-99f1-792ce90dcfcf",
+      "title": "",
+      "user_permissions": [],
+      "username": "test+convergence@inclusion.beta.gouv.fr"
```

Et je proposerais bien également d'arrêter de mettre un email histoire de plus facilement identifier d'où vient une information affichée (est-ce que je vois l'email ou l'username de l'utilisateur dans ce message d'erreur ?).
Ici on pourrait mettre

```suggestion
      "username": "convergence_test_user"
```

(ou 
```suggestion
      "username": "institution_test_user"
```

---

Il y a pas mal des autres utilisations des emails dans les noms d'utilisateur de cette fixture mais je les ai pas changé pour l'instant pour merger plus rapidement ce PR. On pourrait les nettoyer à la prochaine